source/chapters/user_interface.rst

```diff
@@ -602,6 +602,10 @@ control rate changes also from your computer's keyboard, see the chapter
   the Pitch/Rate slider positioned at the center, the pitch rate display is at
   +0.0%, which indicates no change.
 
+.. hint:: To make the rate and/or pitch automatically reset whenever a new
+          track is loaded go to :menuselection:`Preferences --> Decks -->
+          Speed (Tempo) and Key (Pitch) options --> Reset on track load`.
+
```

Comment:
the indentation has to match the above paragraph = +2 spaces

Reply:
oh, sorry, my hint was unclear obviously.
Indentation inside the hint block was fine, just the entire block should be at the same i. level as the paragraph above.

Reply:
actually, pre-commit would notice that if it wouldn't fail earlier.